lib/frizz/remote.rb

```diff
@@ -39,7 +50,9 @@ def delete(remote_file)
     attr_reader :bucket_name, :ignorance, :options
 
     def ignore?(object)
-      ignorance.ignore?(object.key)
+      # for remote files, you don't need to ignore for gzip preference,
+      # since there won't be any gzip files on s3
+      ignorance.ignore_matched_pattern?(object.key)
```

Comment:
We just need `Ignorance` to have the ability to operate on either a full-path or a local-path, right? What if `ignore?` could receive a "path-base" or something, that could be joined with the local path to get the full path..

```ruby
def ignore(local_path, base_path = nil)
  full_path = File.join(base_path, local_path)
  ignore_matched_pattern(path) || ignore_file_for_gzip_version?(full_path)
end
```

That way we give `ignore?` the ability to ignore based on either full or local paths, without tying us to this single use case (gzip) in its interface.

Thoughts?